src/utils.py

```diff
@@ -179,7 +179,7 @@ def get_info(cls, search_key):
         except NoResultFound:
             logger.info("No info for search_key '%s' was found", search_key)
             return {'errnoror': 'No information in the records', 'is_valid': False}
-        except SQLchemyError:
+        except SQLAlchemyError:
             session.rollback()
```

Comment:
would be nice to log this exception as well

Reply:
@GeetikaBatra I think what Pavel meant here was something like:
```python
logger.exception('Error in storing the record in current session')
raise
```

Reply:
@msrb, currently, I was raising all the exceptions in utils which were being catched in res_api. So, didn't logged these exceptions, instead I just raised them. Would you like me to log all the exceptions in utils?
@tisnik  ^